.github/workflows/action-semantic-pull-request.yml

```diff
@@ -0,0 +1,27 @@
+name: "Semantic PR"
```

Comment:
Let's move this file here: https://github.com/mpetrunic/github-mgmt/tree/feat/js-libp2p-semantic-prs/files/.github/workflows

The difference is that `.github/workflows` is a directory for workflows that are to be executed in this repo, while `files/.github/workflows` is intended for the workflow files that we want to "copy" to other repos.